internal/pkg/call/call_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package call
+
+import (
+	"go/ast"
+	"go/importer"
+	"go/parser"
+	"go/token"
+	"go/types"
+	"testing"
+
+	"golang.org/x/tools/go/ssa"
+	"golang.org/x/tools/go/ssa/ssautil"
+)
+
+const source = `package test
+func NoArgs() {}
+func OneArg(one int) {}
+func CallNoArgs() {
+	NoArgs()
+}
+func CallOneArg() {
+	OneArg(0)
+}
+`
+
+type fakeReferrer struct {
+	hasPathTo bool
+}
+
+func (f fakeReferrer) HasPathTo(node ssa.Node) bool { return f.hasPathTo }
+
+func TestRegularCallWithNoArgsIsNotReferredBy(t *testing.T) {
```

Comment:
nit: Please take a look at table-driven-test.
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests
It is the preferred method both in k8t and in google3.

Reply:
Sounds good, I've merged it with the existing table driven test. I think the result is clearer!